Form-Controls/styles.css

```diff
@@ -0,0 +1,83 @@
+/* General */
+
+html {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+*,
+*:before,
+*:after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+/* main */
+
+main {
+  font-size: 20px;
+  margin: 3rem 20rem;
```

Comment:
This margin is pretty large. Would it make sense to set a max-width on this content and margin: auto? Why might I be suggesting this change?